app/components/TermsCheckbox/styled.tsx

```diff
@@ -0,0 +1,10 @@
+// @ts-ignore
+import styled from 'styled-components';
+
+import Box from 'components/Box';
+
```

Comment:
Let's add //@ts-ignore here to hide en error 
<img width="1255" alt="image" src="https://github.com/Michigan-State-University/cias-web/assets/86963976/c0c14894-83fd-49bf-b7f6-f96b5e5ffefb">


```suggestion

// @ts-ignore
```